src/app/routine-home/routine-home.page.html

```diff
@@ -55,8 +44,13 @@ <h5>
     </ion-item>
   </div>
 
+    <h4 class="text-center text-primary-default">
+      My Self Care Goal
+    </h4>
+
   <ion-text color="#B7472A">
-    <h5>My Self Care Goal</h5>
+    <h5 class="text-center mt-2 ml-5 mr-5 mb-10">I would like to achieve work/life balance and feel more energetic daily.
+    </h5>
   </ion-text>
 
   {{data.routine_goal}}
```

Comment:
This is the actual text (as a variable right now) that you want to style I think. You can test by adding a routine goal through the Edit page to see it.

Reply:
I helped Ming Shi with this change. I will approve.